css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,66 @@
+body {
+  font-family: "DM Sans", sans-serif;
+}
+
+.container {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+}
+
+.align-bottom {
+  align-items: end;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-fifth {
+  display: flex;
+  flex-direction: column;
```

Comment:
You should apply these on a separate rule, `column-` class rulesets should only include `flex-basis` or `width` declarations.